packages/ui/src/components/organisms/Form/DynamicForm/fields/EntityFieldGroup/hooks/useEntityFieldGroupList/useEntityFieldGroupList.ts

```
@@ -0,0 +1,72 @@
+  }, [value, onChange]);
+
+  const removeItem = useCallback(
+    async (id: string) => {
+      if (!Array.isArray(value)) {
+        return;
+      }
+
+      // const entity = value.find(entity => entity.__id === id);
+
+      // if (entity?.id) {
+      if (true) {
```

Biome (1.9.4)

[error] Unexpected constant condition.



(lint/correctness/noConstantCondition)